Cargo.lock

```diff
@@ -188,6 +188,16 @@ dependencies = [
  "libc",
 ]
 
+[[package]]
+name = "bzip2"
```

Comment:
Are there bzip'd python packages? Never came across any

Reply:
https://www.python.org/dev/peps/pep-0527/#file-extensions

> Currently sdist supports a wide variety of file extensions like .tar.gz, .tar, .tar.bz2, .tar.xz, .zip, .tar.Z, .tgz, and .tbz. However, of those the only extensions which get anything more than negligible usage is .tar.gz with 444,338 sdists currently, .zip with 58,774 sdists currently, and .tar.bz2 with 3,265 sdists currently.

It's deprecated, I'm going to move deprecated formats behind a off-by-default feature in `python-pkginfo` crate later.